app/src/components/market/common/arbitrators/index.tsx

```diff
@@ -14,12 +14,15 @@ const Wrapper = styled.div`
 `
 
 const ArbitratorButton = styled(Button)<{ isSelected: boolean }>`
-  margin: 0 8px 12px 0;
+  margin: 0 12px 12px 0;
   &,
   &:hover {
     border-color: ${props =>
       props.isSelected ? props.theme.textfield.borderColorActive : props.theme.buttonSecondaryLine.borderColor};
   }
```

Comment:
I think this section should be outside of `:hover`

Reply:
done